JellyfinPlayer/Components/PillHStackView.swift

```diff
@@ -0,0 +1,60 @@
+//
+ /* 
+  * SwiftFin is subject to the terms of the Mozilla Public
+  * License, v2.0. If a copy of the MPL was not distributed with this
+  * file, you can obtain one at https://mozilla.org/MPL/2.0/.
+  *
+  * Copyright 2021 Aiden Vigue & Jellyfin Contributors
+  */
+
+import SwiftUI
+
+protocol PillStackable {
+    var title: String { get }
+}
+
+struct PillHStackView<NavigationView: View, ItemType: PillStackable>: View {
+    
+    let title: String
+    let items: [ItemType]
+    let navigationView: (ItemType) -> NavigationView
+    
+    var body: some View {
+        VStack(alignment: .leading) {
+            Text(title)
+                .font(.callout)
+                .fontWeight(.semibold)
+                .padding(.top, 3)
+                .padding(.leading, 16)
+            
+            ScrollView(.horizontal, showsIndicators: false) {
+                HStack {
+                    ForEach(items, id: \.title) { item in
```

Comment:
Wouldn't it be good to replace VStack and HStack that use `ForEach` with LazyStack and LazyHStack?
If there's no layout issue

Reply:
There are many differences in performance with and without `Lazy`, even with simple-structured views. Because `HStack` and `VStack` loads all inner views when load a parent View.